postgres_exporter/common/pg10/queries_general.yml

```diff
@@ -53,11 +53,16 @@ ccp_replication_lag:
            WHEN pg_last_wal_receive_lsn() = pg_last_wal_replay_lsn() THEN 0
            ELSE EXTRACT (EPOCH FROM now() - pg_last_xact_replay_timestamp())::INTEGER
            END
-        AS replay_time"
+        AS replay_time
+        , EXTRACT (EPOCH FROM now() - pg_last_xact_replay_timestamp())::INTEGER
+        AS received_time"
   metrics:
     - replay_time:
         usage: "GAUGE"
-        description: "Length of time since the last transaction was replayed on replica. Will always increase if no writes on primary."
+        description: "Length of time since the last transaction was replayed on replica. Returns zero if last WAL recieved equals last WAL replayed. Avoids false positives when primary stops writing. Monitors for replicas that cannot keep up with primary WAL generation."
+    - received_time:
+        usage: "GAUGE"
+        description: "Length of time since the last WAL file was received and replayed on replica. Always increases, even if primary stops writing. Monitors for replicas that stop receiving WAL all together."
```

Comment:
```suggestion
        description: "Length of time since the last WAL file was received and replayed on replica. Always increases, causing false positives if the primary stops writing. Monitors for replicas that stop receiving WAL all together."
```

Reply:
ditto for the other copies of this

Reply:
Not sure if I like the wording of this here. For someone not familiar with it, might kind of imply that it's always a false positive if they set up an alert using it.

Reply:
Add a "possibly" in there and I think that works and can just pull your change in. Realized you were going of the description of the other metric.

Reply:
Will pull this change in and change wording slightly.